WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -403,6 +403,7 @@ internal static ImmutableList<SmartCoin> SelectCoinsForRound(IEnumerable<SmartCo
 		var filteredCoins = coins
 			.Where(x => parameters.AllowedInputAmounts.Contains(x.Amount))
 			.Where(x => parameters.AllowedInputTypes.Any(t => x.ScriptPubKey.IsScriptType(t)))
+			.Where(x => !x.IsBanned)
```

Comment:
I think this should be filtered out earlier in CoinJoinManager, where we filter unconfirmed coins as well.